mp/src/game/client/momentum/ui/MapSelection/MapFilterPanel.cpp

```diff
@@ -148,6 +150,11 @@ void MapFilterPanel::OnCommand(const char* command)
         m_pResetFiltersButton->SetEnabled(false);
         m_pApplyFiltersButton->SetEnabled(false);
     }
+    else if (FStrEq(command, "FeelingLucky"))
+    {
+        MapSelectorDialog().StartRandomMapFromCurrentTab();
+        m_pFeelingLuckyButton->SetEnabled(true);
```

Comment:
Did you mean to set `true` here? We should disable the button until the random map either starts or fails to start. Downloading could take a bit and we don't want users to click it again or switch tabs and click it again.

Reply:
This might mean making the `m_uStartMapWhenReady` part of the dialog rather than the BaseMapsPage.